Add introduction of a few concepts (#39)
OpenMS/OpenMS-docs: docs/introduction.md

spectrometry related software. OpenMS is free software available under the
three clause BSD license and runs under Windows, macOS, and Linux.

```{note}

> Note: This introduction is aimed at users new to the field of LC-MS data analysis and will introduce some basics terms

Reviewer: Will this be fine on sphinx or will it be double Note Note:?

and concepts. How to handle the data analysis, available data structures, algorithms and more are covered in the various
subsections of this documentation.
```

# Background

Proteomics and metabolomics are interdisciplinary research fields that study structure, function, and interaction of

> proteins and metabolites. They employ large-scale experimental techniques that allow acquiring data at the level of

Reviewer 2: break mid-sentence

cellular systems to whole organisms. One of the main analytical method to identify, characterize or quantify proteins
and metabolites is mass spectrometry (MS) combined with chromatographic separation.

> 

Reviewer 2: Mass spectrometry combined with chromatographic separation is commonly used to identify, characterize or quantify the amount of proteins and metabolites.

In mass spectrometry-based proteomics and metabolomics, biological samples are extracted, prepared, and separated to
reduce sample complexity. The separated analytes are ionized and measured in the mass spectrometer. Mass and abundance

> of ions are stored in mass spectra and used to identify and quantify the analytes in the sample using computational

Reviewer 2: break mid-sentence

methods. The quantity and identity of analytes can then be used, for instance, in biomarker discovery, medical diagnostics,
or basic research.

# Liquid Chromatography

> LC aims to reduce the complexity of the measured sample by separating analytes based on their physicochemical properties.

Reviewer 2: Liquid Chromatography (LC)
or Liquid chromatography

Separating analytes in time ensures that a manageable amount of analytes elute at the same time. In mass
spectrometry-based proteomics, (high-pressure) liquid chromatographic separation techniques (HPLC) are methods of choice
to achieve a high degree of separation. In HPLC, peptides are separated on a column. Solved in a pressurized liquid
(mobile phase) they are pumped through a solid adsorbent material (stationary phase) packet into a capillary column.
Physicochemical properties of each peptide determine how strongly it interacts with the stationary phase. The most

> commonly HPLC technique in proteomics and metabolomics uses reversed-phase chromatography (RPC) columns. RPC employs a

Reviewer 2: The most common

Reviewer 2: The most common HPLC technique in proteomics and metabolomics use reversed-phase chromatography (RPC) columns.

  of an ion allows calculating the velocity which in turn is used to calculate the mass-to-charge ratio (m/z). Varying
  the electric field allows filtering certain mass-to-charge ratios before they enter the detector. In quadrupole mass
  filters, ions pass through an oscillating electric field created by four parallel rods. For a particular voltage, only
  ions in a certain mass-to-charge range will reach the detector. The orbitrap is an ion trap mass analyzer (and detector)
  that traps ions in orbital motion between a barrel-like outer electrode and a spindle-like central electrode allowing
  for prolonged mass measurement. As a result of the prolonged mass measurements, a high mass resolution can be achieved.

- **Detector**: The last component of the mass spectrometer is the detector. It determines the abundance of ions that
  passed through the mass analyzer. Ion intensities (a value that relates to its abundance) and the mass-to-charge ratio
  are recorded in a mass spectrum.

A sample is measured over the retention time of the chromatography typically resulting in tens of thousands of spectra.

> The measurement of one sample is called an MS run and the set of spectra called an MS or peakmap.

Reviewer: ```suggestion
The measurement of one sample is called an MS run and the set of spectra called an MS(1) map or peak map.
```

![](images/introduction/spectrum_peakmap.png)

> The left image displays spectrum with peaks (m/z and intensity values) and the right image shows a spectrum stacked in

Reviewer: ```suggestion
The left image displays spectrum with peaks (m/z and intensity values) and the right image shows spectra stacked in
```

> Left image displays spectrum with peaks (m/z and intensity values) and right image shows spectra stacked in retention

Reviewer 2: The left image

Reviewer 2: and the right image

Reviewer: And "a spectrum"

> retention time yield a peak map.

Reviewer: ```suggestion
retention time yielding a peak map.
```

In proteomics and metabolomics, the MS1 intensity is often used for the quantification of an analyte. Identification
based on the MS1 mass-to-charge and the isotope pattern is highly ambiguous. To improve identification, tandem mass
spectrometry (MS/MS) can be applied to assess the analyte substructure. To this end, the precursor ion is isolated and
kinetically fragmented using an inert gas (e.g., Argon). Fragments produced by collision-induced fragmentation (CID) are

> stored in an MS2 (MS/MS) spectrum and provide information that helps to resolve the ambiguities in identification.

Reviewer: ```suggestion
stored in an MS2 (or MS/MS) spectrum and provide information that helps to resolve the ambiguities in identification.
```
